commit/merkleroot/types.go

```diff
@@ -73,10 +80,16 @@ func aggregateObservations(aos []plugincommon.AttributedObservation[Observation]
 				append(aggObs.OffRampNextSeqNums[seqNumChain.ChainSel], seqNumChain.SeqNum)
 		}
 
+		// RMNRemoteConfig
+		if !obs.RMNRemoteConfig.IsEmpty() {
+			aggObs.RMNRemoteConfigs = append(aggObs.RMNRemoteConfigs, obs.RMNRemoteConfig)
```

Comment:
Not sure if we do this elsewhere but where do we validate that RMNRemoteConfig is only fetched by dest readers?